knife-ec-backup.gemspec

```
@@ -17,7 +17,7 @@ Gem::Specification.new do |s|
   # ... but, we can live with those options not working, if it means the plugin
   # can be included with apps that have restrictive Gemfile.locks.
   # s.add_dependency "mixlib-cli", ">= 1.2.2"
-  s.add_dependency "sequel"
+  s.add_dependency "sequel", "~> 5.9.0"
```

You might want to pin this a bit less strict
```suggestion
  s.add_dependency "sequel", "~> 5.9"
```